python/spear/game_world_service.py

```diff
@@ -2,9 +2,92 @@
 # Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
 #
 
+from enum import Enum
 import json
 
 class GameWorldService():
+
+    class PropertyDesc():
```

Comment:
add comment that this enum needs to match its C++ counterpart

Reply:
See my most recent design sketch to avoid duplicating most of these enums in Python.